lib/elixir_google_scraper_web/controllers/api/v1/keyword_controller.ex

```diff
@@ -41,6 +42,18 @@ defmodule ElixirGoogleScraperWeb.Api.V1.KeywordController do
     end
   end
 
+  def show(conn, %{"id" => id}) do
```

Comment:
Let's check if keyword is belong to the requested user

Reply:
Done, API will respond 404 error if the keyword does not exist in the database or keyword does not belong to the user